CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ Changelog
 -----
 
 ### Enhancements
-  
+
 - Add support for sending "breadcrumbs" - notable events leading up to an error
   [Christian Schlensker](https://github.com/wordofchristian)
   [#149](https://github.com/bugsnag/bugsnag-js/pull/149)
 
-  
+### Deprecations/Breakages
```

Comment:
> Do we have a more standardized header for this kind of thing?

I’ve been working it into a summary paragraph before the sections. [This is maybe an example](https://github.com/bugsnag/bugsnag-cocoa/blob/master/CHANGELOG.md#500), as is [this](https://github.com/bugsnag/bugsnag-php/blob/master/CHANGELOG.md#300-2016-07-07). If we need some instructions for how to migrate, I’d recommend an UPGRADING.md similar to [bugsnag-php](https://github.com/bugsnag/bugsnag-php/blob/master/UPGRADING.md)